Blanball/Views/CommonButton/CommonButton.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  CommonButton.swift
+//  Blanball
+//
+//  Created by Nik Dub on 09.03.2023.
+//
+
+import UIKit
+
+class CommonButton: UIButton {
+    
+    //TODO: Implement another solution or enhance current
+    
+    // MARK: - Private properties -
+    
+    override var isEnabled: Bool {
+        didSet {
+            backgroundColor = isEnabled
+            ? switchedBackgroundColor.0
+            : switchedBackgroundColor.1
+        }
+    }
+    
+    private var switchedBackgroundColor: (UIColor?, UIColor?) = (nil, nil) {
```

Comment:
example of naming: backgroundColors: (onColor: UIColor?, offColor: UIColor?) = (nil, nil)

Reply:
New names:

> backgroundColors, borderColors,  tintColors

`Changes According PR#7`